nettacker/core/graph.py

```diff
@@ -179,25 +274,34 @@ def create_report(options, scan_id):
             + "</p>"
             + log_data.json_parse_js
         )
-        with open(report_path_filename, "w", encoding="utf-8") as report_file:
+        with Path(report_path_filename).open("w", encoding="utf-8") as report_file:
             report_file.write(html_table_content + "\n")
-            report_file.close()
+
+    elif len(report_path_filename) >= 5 and report_path_filename[-8:].lower() == ".dd.json":
+        with Path(report_path_filename).open("w", encoding="utf-8") as report_file:
+            dd_content_json = create_dd_specific_json(all_scan_logs)
+            report_file.write(dd_content_json + "\n")
```

Comment:
yes, this is consistent with the existing code, just have to mention that `+` strings in Python is a no-no.